soc/arm/microchip_mec/mec172x/soc.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __MEC_SOC_H
+#define __MEC_SOC_H
+
+#define SYSCLK_DEFAULT_IOSC_HZ MHZ(96)
```

Comment:
What does this actually do?

Reply:
Another addition I did not make and don't know what it is for.  Can't find it with git grep or manual search. Doesn't cause build failure so I'll remove it.